### PR TITLE
User migration

### LINKED ISSUE #2281: User data rework
Rename fields 'surname' and 'name' in 'firstname' and 'lastname' for user.

### PR DESCRIPTION
**Pull request by Mussuranae**
Change on user collection : firstname and lastname instead of name and surname, to get more consistency in the DB
Fix #2281

### CODE REVIEW

File: apps/backend-ops/src/firestoreMigrations/0014.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * Update name and surname in user documents
+ */
+export async function upgrade(db: Firestore) {
+  const users = await db.collection('users').get();
+
+  const newUserData = users.docs.map(async (userDocSnapshot: any): Promise<any> => {
+    const userData = userDocSnapshot.data();
+    const { name, surname } = userData;
+
+    if (userData.name && userData.surname) {
```
Comment (phpgeekfr) [inline apps/backend-ops/src/firestoreMigrations/0014.ts]: What if only one is defined ? We will still keep the other one on DB.
You should check theme separatly

File: libs/organization/src/lib/organization/+state/organization.firestore.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ interface OrganizationRaw<D> {
   cart: CatalogCart[];
   created: D;
   denomination: Denomination;
-  description: string;
+  description?: string;
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/+state/organization.firestore.ts]: why this change ? (update draw.io if need to keep this)

File: libs/organization/src/lib/organization/+state/organization.firestore.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ interface AppAccess {
 
 interface Denomination {
   full: string;
-  publicName?: string;
+  public?: string;
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/+state/organization.firestore.ts]: Don't forget to update draw.io

Comment (phpgeekfr) [reply libs/organization/src/lib/organization/+state/organization.firestore.ts]: and migrations ?

File: libs/organization/src/lib/organization/+state/organization.firestore.ts (hunk 2 above)
Comment (Mussuranae) [reply libs/organization/src/lib/organization/+state/organization.firestore.ts]: As no one org has a description currently, I though to let this one no mandatory for now, or I create one by default for every organization.

File: libs/organization/src/lib/organization/+state/organization.firestore.ts (hunk 3 above)
Comment (Mussuranae) [reply libs/organization/src/lib/organization/+state/organization.firestore.ts]: It's because I put only "public" on the migration that I change that in this PR -_- My bad for this one

File: apps/backend-ops/src/firestoreMigrations/0014.ts (hunk 1 above)
Comment (Mussuranae) [reply apps/backend-ops/src/firestoreMigrations/0014.ts]: Sure ! I will update that !
